Geometry/MTDNumberingBuilder/src/GeometricTimingDet.cc

```diff
@@ -5,7 +5,7 @@
 #include "CondFormats/GeometryObjects/interface/PGeometricTimingDet.h"
 
 #include "CLHEP/Units/GlobalSystemOfUnits.h"
```

Comment:
Please delete this line 7. It should have been done long ago. Also requiring change is line 232:
```
Position pos(float(trans_.x() / cm), float(trans_.y() / cm), float(trans_.z() / cm));
```
It will become:
```
Position pos(float(trans_.x() / dd4hep::cm), float(trans_.y() / dd4hep::cm), float(trans_.z() / dd4hep::cm));
```

Reply:
Just to confirm: As of now `cm` is 10. and `dd4hep::cm` is 1. Therefore, applying the suggested change we would expect these quantities to increase by a factor of 10. and not remain the same. Is that correct?

Reply:
Sorry, I goofed. In CLHEP `/ cm` means "convert from mm to cm", so the line should become:
```
Position pos(static_cast<float>(convertMmToCm(trans_.x()), static_cast<float>(convertMmToCm((trans_.y())), static_cast<float>(convertMmToCm((trans_.z())));
```
I don't think this line is related to DD4hep. It's confusing. The translation is initially converted to mm and then converted to cm for this position.
```

Reply:
Ok. I think is done now :+1: